source/draw/RasterYuvSemiplanar.ooc

```diff
@@ -50,7 +50,9 @@ RasterYuvSemiplanar: abstract class extends RasterPlanar {
 		this uv referenceCount decrease()
 		super()
 	}
-
+	apply: override func ~rgb (action: Func (ColorRgb)) {
+		this apply(ColorConvert fromYuv(action))
```

Comment:
Maybe that should be fixed in the compiler since C does not require dynamic memory for function pointers.